www/src/views/showcase/filtered-showcase.js

```diff
@@ -118,15 +119,39 @@ class FilteredShowcase extends Component {
           aggregatedCategories={aggregatedCategories}
         />
         <ContentContainer>
-          <ContentHeader>
+          <ContentHeader
+            cssOverrides={{
+              height: `6.9rem`,
+              flexDirection: `column-reverse`,
+              alignItems: `flex-start`,
+              pt: 5,
+              pb: 4,
+              [mediaQueries.sm]: {
+                height: `4rem`,
+                flexDirection: `row`,
+                alignItems: `center`,
+                pt: 0,
```

Comment:
This can be removed because this is being handled by theme-ui's responsive syntax above
```suggestion

```